### DISCUSSION TITLE
Account paused

### DISCUSSION THREAD

**Discussion by c0rdeiro** [vercel/vercel#11138] [Help]
### Question

So, I had a project that went over the free tier bandwidth but I noticed it too late. 
I ended up creating a Team with Pro and transfered the project there but now my personal account keeps being paused.
The email I received with the 'next steps' just says 'update to pro' but this personal account I usually just do some pet project apps, nothing big so it doesn't make sense to upgrade. How can this be fixed?

**Comment by elchiconube** [comment]
I have the same issue. My personal account was paused but I don't know how to fix it or even why it's been paused. Any advise here? Thanks
